README.adoc

```diff
@@ -36,30 +38,238 @@ include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/
 
 
 [[initial]]
-== Create a resource controller
+== Add required dependencies
```

Comment:
Suggestion:
+== Add the following dependencies
* spring-integration-gcp
* spring-cloud-gcp-starters-pubsub

Remove rest of text until line 47.